lms/services/vitalsource/service.py

```diff
@@ -0,0 +1,32 @@
+from lms.services.vitalsource._client import VitalSourceClient
+from lms.services.vitalsource.model import VSBookLocation
+
+
+class VitalSourceService:
+    """A high-level interface for dealing with VitalSource."""
+
+    def __init__(self, client: VitalSourceClient):
+        self.client = client
+
+    def get_book_info(self, book_id: str):
+        """Get details of a book."""
+
+        return self.client.get_book_info(book_id)
+
+    def get_book_toc(self, book_id: str):
+        """Get the table of contents for a book."""
+
+        return self.client.get_book_toc(book_id)
```

Comment:
These are a bit of a pain, but pretty soon we're probably going to have two clients and it would be good to hide the details from the consumer.

This also makes the public interface very clear. At the moment there are only two methods on the client and they are both exposed, but soon there will be a lot more and they will not all be directly exposed. Instead we will have methods which use two or three of them to get something done.